src/engine/renderer/gl_shader.cpp

```diff
@@ -953,6 +1011,8 @@ void GLShaderManager::PrintInfoLog( GLuint object, bool developerOnly ) const
 	int         maxLength = 0;
 	int         i;
 
+	// TODO: Not sure how to upgrade this type of usage to the
```

Comment:
I'd tranform it to GetInfoLog and let the caller decide where to output the string